source/includes/inventory/introduction.md

```diff
@@ -2,4 +2,12 @@
 
 # Inventory App for retailers
 
-(Redaction in progress)
\ No newline at end of file
+The inventory app is a mobile app (iOS and Android) to process your incoming inventory.
+
+It is available in English and French.
+
+It allows you to track your incoming parcels.
+
+Screenshots below are all made using an iPhone 6. The interface is the same for both platforms.
+
+If you have any question please send an email to : leo_at_shoprunback.com
```

Comment:
I have just created a dedicated mailing list for this, we have to update it on all the documentation material.

The address is tech _at_ shoprunback _dot_ com